### PR TITLE
Refactor openshift controller manager and cluster policy controller

### PR DESCRIPTION
**Pull request by csrwng**
Removes the use of templates and adds reconciliation code for both
controllers.

### CODE REVIEW

File: control-plane-operator/controllers/hostedcontrolplane/cluster_policy/config.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
+package cluster_policy
```
Comment (ironcladlou) [inline control-plane-operator/controllers/hostedcontrolplane/cluster_policy/config.go:1]: Nit: conventional package name would be `clusterpolicy`

**Review by ironcladlou — CHANGES_REQUESTED** [review]
Just one nit about a package name, otherwise looks straightforward to me

File: control-plane-operator/controllers/hostedcontrolplane/cluster_policy/config.go (hunk 1 above)
Comment (csrwng) [reply control-plane-operator/controllers/hostedcontrolplane/cluster_policy/config.go]: thx, updated

**Comment by ironcladlou** [comment]
/lgtm
/hold

Just waiting for e2e

**Pull request merged**